Jo and Astha's team/Shreyash/Flappy_Birds/Flappy_Birds.pde

```diff
@@ -0,0 +1,134 @@
+PImage bg,go;
+drogo player;
+ArrayList<Logs> log = new ArrayList<Logs>();
+power pow_1;
+boolean flag=true;
+
+
+
+int bgx;
+boolean ch=true,inv=true;
+int score=0,total_s=0;
+void setup(){
+   size(720,405,P2D);
+   bg = loadImage("./Image/bg.png");
+   player = new drogo();
+   log.add(new Logs());
+   pow_1=new power();
+   frameRate(60);
+   
+}
+
+void draw(){
+  
+  if(ch)
+  {
+    back();
+    player.display();
+    player.canvas();
+    String sc="Your Score- ";
+     text(sc,850,50);
+     
+    for(int i=0;i<log.size();i++){
+       log.get(i).display();
+       
+       if(inv)
+       {
+         if(!log.get(i).overlap(player)){
+           log.clear();
+           ch=false;
+         }
+         else if(log.get(i).check()&&flag)
+         {
+           flag=false;
+           if(score<22)
+               {
+                 score++;
+                 total_s++;
+               }
+         }
+       }
+       else
+       {
+         total_s++;
+         if((frameCount/60)%15==1)
+         {
+           inv=true;
+         }
+         score=0;
+       }
+       
+     }
+     
+    pow_1.display(score);
+      PFont f = createFont("impact",20);
+      String s = "Your Score- "+total_s;
+      textFont(f);
+      fill(0);
+      text(s,600,20);
+      
+    while(score>=22){
```

Comment:
```suggestion
    if(score>=22){
```
`while` loop makes it much heavier task, changing it to `if` should make it a lot smoother